scripts/apps/vocabularies/components/UploadConfigModalInformation.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import {gettext} from 'core/utils';
+
+export const UploadconfigModalInformation = ({label}) => {
+    return (
+        <div className="sd-alert sd-alert--hollow sd-alert--primary">
+            <span>
+                <i className="big-icon--info big-icon--blue" style={{margin: '0 0.6rem 0 -0.3rem'}} />
+            </span>
+            <span>
+                {gettext(label)}
```

Comment:
Only inline strings can be passed to `gettext`, not variables. Drop `gettext` here and make sure that the value that comes from props is already translated in the same place where it's defined.